### DISCUSSION TITLE
Fabrication and Expansion update for 1.15+

### DISCUSSION THREAD

**Discussion by jordan8396** [MrTJP/ProjectRed#1722] [Q&A] [answered]
Hello, just wondering whether there is an update on Fabrication and Expansion modules being brought to versions 1.15+?

More specifically for 1.16.5 - as unfortunately this is the version my map had updated to before I ported it from xbox to mod it 😑
Loved using the modules on 1.12.2 and would love to have that versatility once again.
Thanks! 😊

**Comment by MrTJP** [comment, marked as answer]
Fabrication is getting the most attention right now. I've been putting all my free time over the last few weeks on it. Almost ready to show off the UI :)

Expansion will come later but I'm not really planning a complete port. Going to take some time and re-write a bunch of stuff. I may release just the machines and stuff soon, but the pipe system is going to be re-written for sure.

**Reply by jordan8396** [reply]
That's great news.
Thank you for all your great work :)
